fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -263,8 +287,32 @@ public Builder useLegacyScrubber(boolean legacy) {
                 return this;
             }
 
+            /**
+             * Choose a specific "Already Scrubbed" rangeSet prefix. This may be useful for
+             * multiple scrubbing jobs that may not affect each other.
+             * 0 is the backward compatible default, which means no prefix.
+             * @param rangeId a rangeSet prefix
+             * @return this builder
+             */
+            public Builder setRangeId(final int rangeId) {
```

Comment:
It seems like it might be nice from a user perspective for this to be a string instead of an int.